apps/vscode/package.json

```
@@ -960,8 +960,16 @@
           "default": "#40404066",
           "markdownDescription": "CSS color for background of executable code cells on dark themes.\n\n*Note that this color should include an alpha channel so that selections show up against the background.*"
         },
-        "quarto.cells.background.delay": {
-          "order": 22,
+        "quarto.cells.background.useTheme": {
+					"order": 22,
+					"scope": "window",
+					"type": "string",
+					"format": "color",
```

```suggestion
					"type": "boolean",
```

This is more like true vs. false, not a string formatted as a color, like what is show in this example:
https://code.visualstudio.com/api/references/contribution-points#Configuration-example

An example in this file is `"quarto.cells.background.enabled"`.